src/js/components/action.jsx

```diff
@@ -165,7 +165,7 @@ export class ActionComponent extends Component {
                            { buttonText }
                        </button>
                    </div>;
-        } else {
+        } else if (type === 'link' || type === 'buy' && !stripeIntegration) {
```

Comment:
maybe wrap the && part in parenthesis to reduce ambiguity?

`if (type === 'link' || (type === 'buy' && !stripeIntegration)) {`